components/web_server.rst

```diff
@@ -78,28 +78,38 @@ Configuration variables:
 
         # Example configuration entry
         web_server:
-          port: 80
+          port: 80        
```

Comment:
Please strip whitespace

Reply:
Done. I was discussing with @jesserockz yesterday - what license should the esphome-webserver have - I've currently assigned MIT but you might have a better suggestion?

Reply:
MIT is fine for me. I usually do Apache but it doesn't matter.